app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, marks, collection_of_coins]
```

Comment:
The `marks` dictionary is included in both the 'mutable' and 'immutable' lists. Dictionaries are mutable, so it should only be in the 'mutable' list. Remove `marks` from the 'immutable' list.